src/config.c

```diff
@@ -470,31 +472,24 @@ int config_parse_conf(struct config *cfg, const struct conf *conf)
 		cfg->avt.rtp_bw.max *= 1000;
 	}
 
-	if (0 == conf_get(conf, "jitter_buffer_type", &jbtype)) {
-		cfg->avt.video.jbtype = conf_get_jbuf_type(&jbtype);
-		cfg->avt.audio.jbtype = conf_get_jbuf_type(&jbtype);
-		warning("config: jitter_buffer_* config is deprecated, use "
-			"audio_jitter_buffer_* and "
-			"video_jitter_buffer_* options\n");
-	}
-
-	(void)conf_get_range(conf, "jitter_buffer_delay",
-			     &cfg->avt.video.jbuf_del);
-	(void)conf_get_range(conf, "jitter_buffer_delay",
-			     &cfg->avt.audio.jbuf_del);
-
 	if (0 == conf_get(conf, "audio_jitter_buffer_type", &jbtype))
 		cfg->avt.audio.jbtype = conf_get_jbuf_type(&jbtype);
 
-	(void)conf_get_range(conf, "audio_jitter_buffer_delay",
+	(void)conf_get_range(conf, "audio_jitter_buffer_delay_ms",
```

Comment:
Maybe a little bit shorter? e.g.:
- audio_jitter_buffer_ms
- audio_jbuf_ms

Similar for video.